unlock-app/src/middlewares/storageMiddleware.js

```diff
@@ -0,0 +1,57 @@
+import {
+  CREATE_LOCK,
+  UPDATE_LOCK,
+  ADD_LOCK,
+  LOCK_DEPLOYED,
+} from '../actions/lock'
+import Web3Service from '../services/web3Service'
+import * as storageMiddlewareHandler from './storageMiddlewareHandler'
+
+export default function storageMiddleware({ getState, dispatch }) {
+  const web3Service = new Web3Service()
```

Comment:
Can we somehow put this in the existing middleware so we reuse the connection to web3Service?

Reply:
This should now be addressed